projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-following.php

```diff
@@ -0,0 +1,105 @@
+<?php
+/**
+ * REST API endpoint for the Jetpack author recommendations block.
+ *
+ * @package automattic/jetpack
+ * @since $$next-version$$
+ */
+
+use Automattic\Jetpack\Connection\Client;
+use Automattic\Jetpack\Status\Visitor;
+
+/**
+ * Class WPCOM_REST_API_V2_Endpoint_AI
+ */
+class WPCOM_REST_API_V2_Endpoint_Following extends WP_REST_Controller {
+	/**
+	 * Namespace prefix.
+	 *
+	 * @var string
+	 */
+	public $namespace = 'wpcom/v2';
+
+	/**
+	 * Endpoint base route.
+	 *
+	 * @var string
+	 */
+	public $rest_base = 'following';
+
+	/**
+	 * WPCOM_REST_API_V2_Endpoint_AI constructor.
+	 */
+	public function __construct() {
+		$this->wpcom_is_wpcom_only_endpoint    = true;
+		$this->wpcom_is_site_specific_endpoint = false;
+
+		add_action( 'rest_api_init', array( $this, 'register_routes' ) );
+	}
+
+	/**
+	 * Register routes.
+	 */
+	public function register_routes() {
+		register_rest_route(
+			$this->namespace,
+			$this->rest_base . '/mine',
+			array(
+				array(
+					'methods'             => WP_REST_Server::READABLE,
+					'callback'            => array( $this, 'get_response' ),
+					'permission_callback' => array( $this, 'permission_check' ),
+					'args'                => array(
+						'ignore_user_blogs' => array(
+							'type' => 'boolean',
+						),
+					),
+				),
+			)
+		);
+	}
+
+	/**
+	 * Check if user has edit post permission.
+	 *
+	 * @return true|WP_Error True if the request has access to create items, WP_Error object otherwise.
+	 */
+	public function permission_check() {
+		return current_user_can( 'edit_posts' );
+	}
+
+	/**
+	 * Gets the sites the user is following
+	 *
+	 * @param WP_REST_Request $request Full details about the request.
+	 */
+	public function get_response( $request ) {
+		$ignore_user_blogs = $request->get_param( 'ignore_user_blogs' );
+
+		if ( defined( 'IS_WPCOM' ) && IS_WPCOM ) {
+			require_once WP_CONTENT_DIR . '/lib/wpcom-user-following.php';
```

Comment:
```suggestion
			require_once WP_CONTENT_DIR . '/lib/wpcom-get-user-followed-blogs.php';
```

I think this should be `wpcom-get-user-followed-blogs.php`